modules/xmpp/strophe.jingle.js

```diff
@@ -375,23 +375,17 @@ export default class JingleConnectionPlugin extends ConnectionPlugin {
                     iceservers[j] = temp;
                 }
 
-                if (options.useStunTurn) {
-                    let filter;
-
-                    if (options.useTurnUdp) {
-                        filter = s => s.urls.startsWith('turn');
-                    } else {
-                        // By default we filter out STUN and TURN/UDP and leave only TURN/TCP.
-                        filter = s => s.urls.startsWith('turn') && (s.urls.indexOf('transport=tcp') >= 0);
-                    }
-
-                    this.jvbIceConfig.iceServers = iceservers.filter(filter);
-                }
+                let filter;
 
-                if (options.p2p && options.p2p.useStunTurn) {
-                    this.p2pIceConfig.iceServers = iceservers;
+                if (options.useTurnUdp) {
+                    filter = s => s.urls.startsWith('turn');
+                } else {
+                    // By default we filter out STUN and TURN/UDP and leave only TURN/TCP.
+                    filter = s => s.urls.startsWith('turn') && (s.urls.indexOf('transport=tcp') >= 0);
                 }
```

Comment:
This changes the behavior where p2p were able to use turn UDP, which does not make sense in jvb connection but make sense for p2p, that's why for the jvb connection we leave only turns TCP.

Reply:
No, note how we don't apply the filter there.

Reply:
Aaa now I see it 🤦